src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
```

Comment:
Using absolute positioning for the card title might cause layout issues if the card size or content changes. Consider using relative positioning or flexbox for more responsive design.